.changeset/overview-stale-filter.md

```
@@ -0,0 +1,17 @@
+---
+"@buildinternet/releases": minor
+---
+
+`releases admin overview-list` lists organizations with their overview status. Pass `--stale` to filter to orgs whose overviews need regeneration:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)